### PR TITLE
I moved the copy right span text to the end of the footer content

### PR DESCRIPTION
**Pull request by atmelaku**
I changed the footer class flex direction to column in style css
I centered the footer text in media query

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -183,7 +183,7 @@ header {
 .down-span,
 .down-nav {
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
```
Comment (jpochetedmead) [inline styles.css]: In general (when there is a big screen) I want the .down-span and .down-nav to be a row, not a column.

File: styles.css [hunk 2]
```diff
@@ -266,9 +266,10 @@ h1 {
   .part-one img {
     width: 450px;
   }
-  .footer {
```
Comment (jpochetedmead) [inline styles.css]: How does deleting .footer properties, and adding these new properties to the .down-span will improve what I asked you needed to be done at the footer of the page when the screen is less than 910px ?

**Comment by jpochetedmead** [comment]
I have closed this Request, but thank you for trying to help :)
If you find anything wrong with my project and have some fix feel free to submit and I'll review it.

File: styles.css (hunk 2 above)
Comment (atmelaku) [reply styles.css]: oh usually, the copy right text should be at the bottom of  the footer content.

**Pull request closed without merging**